apiserver/facades/controller/caasoperatorprovisioner/provisioner.go

```diff
@@ -133,27 +136,33 @@ func (a *API) OperatorProvisioningInfo(args params.Entities) (params.OperatorPro
 		names.NewControllerTag(cfg.ControllerUUID()),
 		modelConfig,
 	)
-
-	imageRepo, err := docker.NewImageRepoDetails(cfg.CAASImageRepo())
+	modelImage, err := a.broker.GetModelOperatorDeploymentImage()
 	if err != nil {
-		return result, errors.Annotatef(err, "parsing %s", controller.CAASImageRepo)
+		return result, errors.Annotatef(err, "getting model operator deployment image")
 	}
-	registryPath, err := podcfg.GetJujuOCIImagePath(cfg, vers)
+
+	modelImageRepoNamespace, err := podcfg.RecoverRepoFromOperatorPath(modelImage)
```

Comment:
Just call this `repo` or `modelImageRepo`

Reply:
refactored with thanks ian!